src/xenia/apu/xma_context.h

```diff
@@ -134,7 +134,7 @@ class XmaContext {
  public:
   static const uint32_t kBytesPerPacket = 2048;
   static const uint32_t kBitsPerPacket = kBytesPerPacket * 8;
-  static const uint32_t kBitsPerHeader = 33;
+  static const uint32_t kBitsPerHeader = 32;
```

Comment:
If we have information on the contents of the header, it would be useful to describe the meaning of the bits, or to refer to some place in the code where the header is already documented some way.

Reply:
32. 0x20. A nice round power of 2. Makes sense. Is that all that's holding this PR back?

Reply:
Meaning of these bits is described here: https://github.com/xenia-project/xenia/blob/master/src/xenia/apu/xma_helpers.h

However there is a small regression due to that PR in https://github.com/xenia-project/game-compatibility/issues/293 which causes it to stuck on loading screen and that is why it waits